src/seat/pointer/mod.rs

```diff
@@ -119,6 +119,13 @@ pub struct PointerData {
     pub(crate) inner: Mutex<PointerDataInner>,
 }
 
+impl PointerData {
+    /// The latest serial from the `Enter` event.
+    pub fn latest_enter_serial(&self) -> Option<u32> {
+       self.inner.lock().unwrap().latest_enter 
```

Comment:
it seems like this isn't indented 4 spaces, causing the lint to fail.

Reply:
Yeah cargo fmt picks that up.